src/V3Randomize.cpp

```diff
@@ -685,7 +685,7 @@ class ConstraintExprVisitor final : public VNVisitor {
         }
         VL_DO_DANGLING(nodep->deleteTree(), nodep);
     }
-    void visit(AstForeach* nodep) override {}
+    void visit(AstBegin* nodep) override {}
```

Comment:
There are visitors here that can be under begins (I think) and will get missed with this, so remove line.

Reply:
It can be changed of course, but this visitor only visits nodes explicitly, to avoid unexpected nodes appearing out of the blue - initially all need to be constraint blocks, and after visiting all are supposed to be only one of those produced by the code, as the `visit(AstNode*)` fallback asserts in line 766:
```c++
    void visit(AstNode* nodep) override {
        nodep->v3fatalSrc(
            "Visit function missing? Constraint function missing for node: " << nodep);
    }
```